controller/search_blackbox_test.go

```diff
@@ -62,6 +66,10 @@ func (s *searchBlackBoxTest) SetupSuite() {
 	s.DBTestSuite.SetupSuite()
 	s.ctx = migration.NewMigrationContext(context.Background())
 	s.DBTestSuite.PopulateDBTestSuite(s.ctx)
+	err := models.Transactional(s.DB, func(tx *gorm.DB) error {
```

Comment:
what about moving this in `DBTestSuite.SetupSuite()` so every test can benefit from it ?

Reply:
@xcoulon it's going to go away soon anyway, so I guess @pranavgore09 should not worry ;)